### ISSUE TITLE
<ins> and <del> HTML tags aren't being read by JAWS like they are in NVDA

### ISSUE THREAD

**Issue by arendalkowski** [FreedomScientific/VFO-standards-support#646]
### Summary

> JAWS screen reader users are complaining that the <"ins> and <"del> (quotes added to stop Git strikethrough) HTML tags aren't being read by JAWS like they are in NVDA, causing users who are blind or have severe low vision to be unable to know when content is inserted and when it is deleted. Whereas our NVDA users are able to know when content is inserted and deleted if the interface is marked up using those HTML tags.
> 
> Example markup:
> <"div>The board shall require"<"del> addition, <"/del><ins>the costs to rise<"/ins>"for the next bidder."
> Currently JAWS does not read the delete and insert actions here but NVDA does.
### Expected result
> What did you expect to happen?
> JAWS should read "The board shall require delete addition insert the costs to rise for the next bidder."

### Actual result
> What actually did happen?
JAWS ignores <"del> and <"ins> tags as if they're CSS or visual only and have no programmatic implications. Which isn't true, those tags have an HTML programmatic relationship to the text and should read by JAWS without needing to gussy up the code for JAWS with aria-label or  <span class="sr-only">start insert</span> to get JAWS to read something.

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...
See code example above...
#### JAWS version and build number
JAWS 2021

 
#### Operating System and version
Windows Version	10.0.19044 Build 19044

#### Browser and version:
Chrome

**Comment by pbossley** [comment]
This test case is here: https://uxtr.wpengine.com/sandbox/ins-del-test.html
But note that this appears to be resolved in V 2022.2206.9 so we can close this request.

**Comment by stevefaulkner** [comment]
closed fixed thanks for info @pbossley
